src/net.h

```diff
@@ -417,6 +419,194 @@ class V1Transport final : public Transport
     size_t GetSendMemoryUsage() const noexcept override EXCLUSIVE_LOCKS_REQUIRED(!m_send_mutex);
 };
 
+class V2Transport final : public Transport
+{
+private:
+    /** Contents of the version packet to send. BIP324 that senders should leave this empty, and
```

Comment:
missing verb after "BIP324":
```suggestion
    /** Contents of the version packet to send. BIP324 stipulates that senders should leave this empty, and
```